### PR TITLE
test: adjust sidebar cypress tests

### PR DESCRIPTION
**Pull request by richbachman**
The sidebar test fails occasionally. Adding a timeout and changing the tests to see if the corresponding disclosure content is visible seems to work better.

I also added a test for the new patterns disclosure.

### CODE REVIEW

File: cypress/integration/e2e/theme-switcher/index.spec.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 describe('Theme switcher', () => {
   beforeEach(() => {
-    cy.visit('/');
+    cy.visit('/', {timeout: 10000});
```
Comment (SiTaggart) [inline cypress/integration/e2e/theme-switcher/index.spec.ts:3]: I don't think we should be waiting 10 seconds before each test. It's also only the sidebar test that is flakey, right? The others were fine without the timeout as far as I know. Do we need these anywhere else?

File: cypress/integration/e2e/sidebar-navigation/index.spec.ts [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
-    cy.get(layoutButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(iconsButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
-    cy.get(contentButton)
-      .click()
-      .should('have.attr', 'aria-expanded', 'true');
+    cy.visit('/', {timeout: 10000});
```
Comment (SiTaggart) [inline cypress/integration/e2e/sidebar-navigation/index.spec.ts:3]: Do we need to revisit the homepage after expanding a section? Can we just do this in a single page visit? 

Waiting 10 seconds between each test seems unnecessarily excessive.

File: cypress/integration/e2e/sidebar-navigation/index.spec.ts [hunk 3]
```diff
@@ -1,37 +1,45 @@
...
+  });
+
+  it('opens the getting started disclosure', () => {
+    cy.get('[data-cy=getting-started-button]').click();
+    cy.get('[data-cy=getting-started-list]').should('be.visible');
+  });
+
+  it('opens the tokens disclosure', () => {
+    cy.get('[data-cy=design-tokens-button]').click();
```
Comment (SiTaggart) [inline cypress/integration/e2e/sidebar-navigation/index.spec.ts:12]: Can we keep the accessibility tests too `.should('have.attr', 'aria-expanded', 'true');` for these please?

File: cypress/integration/e2e/theme-switcher/index.spec.ts (hunk 1 above)
Comment (richbachman) [reply cypress/integration/e2e/theme-switcher/index.spec.ts:3]: I may have misunderstood the `timeout` argument here. I thought it was the time to wait for `cy.visit('/')` to resolve. Not an actual wait 10 seconds. 

I thought the localhost running in the action might be running slow, and not actually resolving in time for the test. Maybe thats not the case. I'll try and bulk these and remove the timeout.

**Comment by TheSisb** [comment]
Test failed again D:

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**